cmd/schema_gen.go

```diff
@@ -13,62 +13,95 @@ import (
 	"github.com/spf13/cobra"
 )
 
+var schemaGenModel string
+
 func init() {
 	rootCmd.AddCommand(schemaGenCmd)
+	// Note: one model, not model filter
+	addModelFlag(schemaGenCmd)
```

Comment:
I think that if we add a `-m` arg, it should act the same as the other  `-m` arguments and allow for loops in schema gen. I'd love to be able do `ddbt schema-gen -m tag:my_models` and schema gen all of them,

Which would involve building the graph so we can apply to the tag, then execute the schema gen function as the graph execute param